sorted_words.rb

```diff
@@ -0,0 +1,18 @@
+#(2) Write a program called sorted_words.rb.  It should prompt the user for words and add each to an array.  
+#The user should be able to add as many words as they like, until they just hit enter to return a blank word. 
+#Then sort the array using the sort method and print it out. 
+
+words = Array.new
+puts "Hi! I can order the list of words. Please write some word: "
+words.push gets.chomp
+
+
+loop do
+    puts "One more (just press 'Enter' if its enough):"
+    word = gets.chomp
+    break if word ==''
```

Comment:
The `break` keyword has a build-in boolean (aka yes/no) condition, so `break if` is like you are using two set of && checking a value, and it'll easily throw errors in a loop.
Programmers prefer to use it(just `break` in it's own line) in a while loop with if/else condition like this: 
```
while true
        puts i * 3
        i += 1
        if i * 3 >= 21
            # Using Break Statement 
            break
        end        
    end
```